pkg/wasm/cache.go

```diff
@@ -157,7 +157,7 @@ func (c *LocalFileCache) Get(downloadURL, checksum string, timeout time.Duration
 		}
 		wasmLog.Debugf("wasm oci fetch %s with options: %v", downloadURL, imgFetcherOps)
 		fetcher := NewImageFetcher(ctx, imgFetcherOps)
-		b, err = fetcher.Fetch(u.Host+u.Path, checksum)
+		b, dChecksum, err = fetcher.Fetch(u.Host+u.Path, checksum)
```

Comment:
If this function does not return error, the checksum should be matched right? So we don't really need to return the checksum, just `dChecksum = checksum` would be good?

Reply:
Need to consider the case where the `sha256` field of the WasmPlugin resource is not set, and the checksum parameter here is an empty string.

Reply:
Ah I see thanks!